src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx

```diff
@@ -144,20 +168,28 @@ const DATAITEM_COLORS_OPTIONS = [
 
 const DataSeriesFormItemModal = ({
   cardConfig,
+  isSummaryDashboard,
   showEditor,
   editDataSeries,
   setEditDataSeries,
   setShowEditor,
   editDataItem,
   setEditDataItem,
+  validDataItems,
   availableDimensions,
   onChange,
   i18n,
 }) => {
   const mergedI18n = { ...defaultProps.i18n, ...i18n };
-  const { id, type } = cardConfig;
+  const { id, type, content } = cardConfig;
   const baseClassName = `${iotPrefix}--card-edit-form`;
 
+  const isTimeBasedCard =
+    type === CARD_TYPES.TIMESERIES ||
+    type === CARD_TYPES.TABLE ||
+    content?.type === BAR_CHART_TYPES.SIMPLE ||
```

Comment:
I think you need to also check for `timeDataSourceId` for the bar charts

Reply:
This check isn't actually that specific. It's just checking for those card types as they are. Not necessarily if they are currently time-based. Just able to be time based.